bigbluebutton-html5/imports/api/2.0/voice-users/server/methods.js

```diff
@@ -2,10 +2,12 @@ import { Meteor } from 'meteor/meteor';
 import mapToAcl from '/imports/startup/mapToAcl';
 import listenOnlyToggle from './methods/listenOnlyToggle';
 import muteToggle from './methods/muteToggle';
+import kickVoiceUser from './methods/kickVoiceUser';
 
 Meteor.methods(mapToAcl(['methods.listenOnlyToggle', 'methods.toggleSelfVoice', 'methods.toggleVoice',
-], {
-  listenOnlyToggle,
-  toggleSelfVoice: (credentials) => { muteToggle(credentials, credentials.requesterUserId); },
-  toggleVoice: muteToggle,
-}));
+  'methods.kickVoiceUser'], {
+    listenOnlyToggle,
+    toggleSelfVoice: (credentials) => { muteToggle(credentials, credentials.requesterUserId); },
+    toggleVoice: muteToggle,
+    kickVoiceUser,
```

Comment:
What are the use case for this method? I don't see it being used anywhere.

Reply:
The use case is to kick voice_only (phone) user, but @antobinary will continue working on that, this case will need more testing.

Reply:
https://github.com/bigbluebutton/bigbluebutton/issues/4457